docs/_includes/checkbox.html

```diff
@@ -46,28 +50,32 @@ <h3 id="checkbox-mixed">Mixed Checkbox</h3>
         <div class="demo__inner">
             <span class="checkbox">
                 <input aria-label="Mixed checkbox example" aria-checked="mixed" class="checkbox__control" type="checkbox" name="checkbox-default" checked />
-                <span class="checkbox__icon" hidden>
-                    <svg class="checkbox__unchecked" focusable="false" height="18" width="18" aria-hidden="true">
-                        {% include symbol.html name="checkbox-unchecked-large" %}
-                    </svg>
-                    <svg class="checkbox__checked" focusable="false" height="18" width="18" aria-hidden="true">
-                        {% include symbol.html name="checkbox-mixed" %}
-                    </svg>
-                </span>
+                <label class="field__label">
```

Comment:
Is the additional layer really necessary? This might require a longer explanation of why we need it (maybe even a walk-through).